docs/scanners/base.md

```diff
@@ -34,3 +34,72 @@ scanner_configs:
       If this does not resolve the error, seek advice from the security team.
       Slack channel: #security
 ```
+
+## Reading/setting up custom configs for your scanner
+
+For setting up custom configurations for your scanner, you can optionally use the helper method ```build_options```.
+You do not have to use this method if it doesn't help you use your scanner. 
+
+For example, let's pretend you intend to send in the configurations in the following format:
+
+```-flag -string=foo --list=foo,bar,baz -bool=true -file=./foobar.js -file_list=foo.js,bar.js -multiple first -multiple second -d```
+
+Note each type for the arguments, the supported types are:
+```ruby
+  :flag
+  :string # Numbers qualify as strings
+  :list # For a list of strings
+  :bool
+  :file 
```

Comment:
can there be more  examples? Do they need to be prepended by `./`? What is the path file relative to?

Reply:
? Those are all the cases ?